SCClassLibrary/Common/Control/GraphBuilder.sc

```diff
@@ -121,6 +134,11 @@ NamedControl {
 		}
 	}
 
+	*newSpec { arg name, values, rate, lags, fixedLag = false, spec;
```

Comment:
I'm just reading now, but you don't return a new instance here, do you?

Reply:
This is superfluous, removing.